### DISCUSSION TITLE
Status?

### DISCUSSION THREAD

**Discussion by collindbell** [InvadingOctopus/octopuskit#13] [General]
Hey, this looks like a fantastic engine and I might adopt it for my early-stage project. Just seeing it's cooled down in the last few months and don't want to adopt abandonware (especially when game controller support is purportedly coming soon!).

**Comment by ShinryakuTako** [comment]
_TL;DR: Not abandoned, maintainer just stuck in meatspace._

Thanks for checking! I am definitely going to continue using this engine for my own game projects, current and next, because I'm not aware of any other engine that uses Swift + SpriteKit. 

Of course there must be other engines that I just haven't discovered yet (OctopusKit itself doesn't readily show up in Google search either) but I can't imagine them being radically better enough for me to abandon OctopusKit, whose main reason for existence is to have an architecture that I personally find intuitive to work with (even though it may be a little inefficient, but it's sufficient for the kind of 2D games I have in mind, so far.)

I'm just preoccupied with various shit in life; MacBook GPU frying up during last year's lockdown, going on a long staycation, getting stuck in crappy AirBnbs that don't provide a proper workspace (typing this while hunched over on my bed), unreliable internet, loved ones passing away, trying to move to a better place, but mostly just sleeping. 😴

As for game controller support, I need to buy a Mac-compatible gamepad first, but it shouldn't be too hard to code it in using Apple's frameworks.

**Reply by collindbell** [reply]
Meatspace is a drag sometimes! Thanks for the quick debrief. Maybe I can chip in on the game controller bit after my project is a bit farther along.

**Reply by davidreedme** [reply]
Got a Ko-fi, Patreon, or even a PayPal link? I’m happy to fund some of whatever you need, @ShinryakuTako.

**Reply by ShinryakuTako** [reply]
@davidreedme [I actually do!](https://www.patreon.com/invadingoctopus) It's in the ReadMe and the Sponsor button. Thank you so much for asking! This is a major encouragement 🥰

Right now I'm mostly just waiting to find a nicer room (have to go to restaurants just to have a table and chair) or better countries to open up for travel.

**Comment by ShinryakuTako** [comment]
In case anyone is still looking at this project, I have been working on a new components-based framework for Godot: https://github.com/InvadingOctopus/comedot

I don't know if the people who were interested in the fate of OctopusKit will see this message, but if you're still looking for a good engine to make games in, give Godot a try! I miss Swift but everything has been so much easier in Godot and it's a blast!
